sleap/gui/app.py

```diff
@@ -1280,7 +1280,9 @@ def _has_topic(topic_list):
             if suggestion_list:
                 labeled_count = 0
                 for suggestion in suggestion_list:
-                    lf = self.labels.get((suggestion.video, suggestion.frame_idx))
+                    lf = self.labels.get(
+                        (suggestion.video, suggestion.frame_idx), use_cache=True
+                    )
```

Comment:
Add coverage for this?

Reply:
I added a test in `test_dataset.py` called `test_suggestions_cache` that ensures the `labels.get((video, index), use_cache=True)` works as expected, but I can create a new test in `test_app.py` that runs through this line specifically.